requirements.txt

```diff
@@ -0,0 +1,154 @@
+appdirs==1.4.4
```

Comment:
I'm assuming this is just an export of the conda environment? Have you given any thought on how to maintain these dependencies?

I'd suggest that we follow a strategy like we do with conda and have the requirements.txt that is committed be unpinned as much as possible and then automatically generate the requirements.txt that is used for building the pypi package.

Reply:
I had very little success in using an unpinned version of this file. Pip ends up trying to install many versions of many packages trying to figure out what will work. I described the process I went through to create it from the conda environment above. In any case we'll need to wait until `tethys_dataset_services` is fully released on PyPI, to have a complete `requirements.txt`

Reply:
Follow up on this in the future. See Issue #846